src/xo-app/remote/index.js

```diff
@@ -0,0 +1,315 @@
+import _, { messages } from 'messages'
+import ActionButton from 'action-button'
+import filter from 'lodash/filter'
+import Icon from 'icon'
+import isEmpty from 'lodash/isEmpty'
+import map from 'lodash/map'
+import React, { Component } from 'react'
+import { connectStore } from 'utils'
+import { error } from 'notification'
+import { injectIntl } from 'react-intl'
+import {
+  createRemote,
+  deleteRemote,
+  disableRemote,
+  enableRemote,
+  subscribeRemotes
+} from 'xo'
+import {format, parse} from 'xo-remote-parser'
+
+const remoteTypes = {
+  local: 'remoteTypeLocal',
+  nfs: 'remoteTypeNfs',
+  smb: 'remoteTypeSmb'
+}
+
+class AbstractRemote extends Component {
+  _enable = id => enableRemote(id)
+  _delete = id => deleteRemote(id)
+  _disable = id => disableRemote(id)
+
+  render () {
+    const {
+      remote
+    } = this.props
+
+    return <tr>
+      <td></td>
+      <td>{remote.name}</td>
+      <td>{this._renderRemoteInfo(remote)}</td>
+      <td>{this._renderAuthInfo(remote)}</td>
+      <td>
+      {remote.enabled &&
+        <span>
+          <span className='text-success'>{this.accessible} <Icon icon='success' />&nbsp;</span>
```

Comment:
Use `{' '}` if you want a space.
ANd you may move it outside of the `<span />` for semantic.